templates/home/api.html

```diff
@@ -21,121 +21,111 @@
 	<dd>Template <!-- Common design patter --></dd>
 
 	<dt>Last review</dt>
-	<dd>2019-05-09</dd>
+	<dd>2022-04-27</dd>
 
 	<dt>Conforming to</dt>
-	<dd>Going to be: Content and IA spec 2.0</dd>
+	<dd>Going to be: Content and IA spec 3.0</dd>
 
 	<dt>Specification</dt>
-	<dd><a href="https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-information-architecture-specification/templates-detailed-specifications/home-page.html">Home page for Canada.ca</a>
+	<dd><a href="https://design.canada.ca/mandatory-templates/home-page.html">Home page for Canada.ca</a>
 </dl>
 
 <h2>Technical specification</h2>
 
-<h3>Home page banner</h3>
+<h3>Your government</h3>
 
 <dl>
 	<dt>Version</dt>
-	<dd>2</dd>
+	<dd>3</dd>
 	<dt>Updated</dt>
-	<dd>2019-05-09</dd>
+	<dd>2022-04-27</dd>
 </dl>
 
-<p>The background image is loaded with the provisional plugin bgimg which set the background image. The background image is hosted in the WMS at this location. <code>https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg</code>.</p>
-
-<p>The intend is to publish background image that represent natural Canadian landscape and may get updated montly or seasonal.</p>
-
-<h4>Previous version</h4>
-
-<details>
-	<summary>Version 1 - 2018-12-27</summary>
-
-	<p>The default background is gray (#eee) without image. The background image is hosted in the WMS at this location. <code>https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg</code>.</p>
-
-	<p>The intend is to publish background image that represent natural Canadian landscape and may get updated montly or seasonal.</p>
-
-</details>
-
-<h3>Promotional banner</h3>
-
-<p>The promotional banner are over, at the bottom, of the home page banner background image.</p>
-
-<dl>
-	<dt>Version</dt>
-	<dd>2</dd>
-	<dt>Updated</dt>
-	<dd>2019-05-09</dd>
-	<dt>Notes</dt>
-	<dd>Provisional color was removed.</dd>
-</dl>
-
-<pre><code>&lt;div class="provisional bg-cover" data-bgimg="https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg"&gt;
-	&lt;div class="container p-0 p-sm-3"&gt;
-		&lt;div class="well header-rwd brdr-0 brdr-rds-0 text-white bg-gctheme"&gt;
-			&lt;h1 property="name" id="wb-cont"&gt;Canada.ca&lt;/h1&gt;
-			&lt;p class="mrgn-tp-md"&gt;The official website of the Government of Canada&lt;/p&gt;
```

Comment:
Just move the deprecated code into an expand/collapse (details/summary HTML element) instead of moving it into a new separated page.

Reply:
done. Thanks